styles.css

```diff
@@ -623,3 +632,453 @@ button:disabled {
+    background-repeat: no-repeat;
+    background-size: auto;
+    position: relative;
+  }
+
+  .non-content {
+    display: none;
+  }
+
+  .custombt {
+    position: absolute;
+    top: 364px;
+    left: 0;
+    height: 58px;
+  }
+
+  .custom-card2 {
+    background-image: url("images/placeholder1.png");
+    background-repeat: no-repeat;
+    background-size: auto;
+    position: relative;
+  }
+
+  .custom-card3 {
+    background-image: url("images/placeholder2.png");
+    background-repeat: no-repeat;
+    background-size: auto;
+    position: relative;
+  }
+
+  .custom-card-shadow {
+    background-image: linear-gradient(180.45deg, #26262600 0.75%, #262626e6 84.18%), url("images/Img Placeholder.png");
+    background-repeat: no-repeat;
+    background-size: auto;
+    position: relative;
+  }
+
+  .portrait {
+    position: absolute;
+    bottom: 8px;
+    gap: 24px;
+    width: 25.62vw;
+    height: auto;
+    padding-top: 0;
+    margin-bottom: 8px;
+    left: 0;
+  }
+
+  .profesional-art {
+    width: 23.4vw;
+    height: auto;
+    margin-left: 16px;
+  }
+
+  .card-description {
+    width: 23.4vw;
+    height: max-content;
+    margin-left: 16px;
+  }
+
+  .tags {
+    width: 16.38vw;
+    margin: 0 0 0 16px;
+  }
+
+  .about-me-section {
+    background: url("images/Frame-footer-desktop.png");
+    background-position: right 5vw;
+    background-repeat: no-repeat;
+    background-size: 48%;
+    background-color: #262626;
+    width: 100vw;
+    height: max-content;
+    align-items: flex-start;
+  }
+
+  .about-me {
+    width: 39.3vw;
+    height: max-content;
+    margin-left: 142px;
+    padding: 0;
+  }
+
+  .about-title {
+    padding: 0;
+    width: 39.3vw;
+    line-height: 82px;
+    color: #f55800;
+    margin-top: 130px;
+  }
+
+  .description-about {
+    width: 39.3vw;
+    font-size: 20px;
+    line-height: 32px;
+    margin: 0;
+    padding: 0;
+  }
+
+  .resume-button {
+    margin-left: 0;
+    margin-bottom: 139px;
+  }
+
+  .solidhr1 {
+    width: 100vw;
+    margin-left: 0;
+    margin-bottom: 0;
+  }
+
+  .skills {
+    flex-direction: row;
+    align-items: flex-start;
+    width: 80.37vw;
+    height: max-content;
+    margin: 0;
+    margin-left: 9.79vw;
+    gap: 10.55vw;
+  }
+
+  .about-me-sectiones {
+    align-items: flex-start;
+    padding: 0;
+    gap: 24px;
+    width: 20.1vw;
+    height: max-content;
+    margin-top: 130px;
+  }
+
+  .about-languages {
+    align-items: flex-start;
+    padding: 0 12px;
+    gap: 8px;
+    width: 186px;
+    height: 216px;
+  }
+
+  .form {
+    flex-direction: row;
+    gap: 24px;
+    width: 80.76vw;
+    height: max-content;
+    margin-left: 9.37vw;
+    align-items: flex-start;
+    padding: 0;
+    margin-bottom: 15.27vw;
+  }
+
+  .primary-text {
+    margin-top: 15.27vw;
+    width: 31.59vw;
+    height: max-content;
+    padding: 0;
+    font-size: 2.77vw;
+    line-height: 3.61vw;
+    color: #262626;
+    text-align: left;
+  }
```

Comment:
- [ ] Please 🙏🏽  use the styles described in the Figma design, especially the font family.

![image](https://user-images.githubusercontent.com/58642949/213597439-f35ae076-ae5e-4468-aefe-c7bd222a1e15.png)